ui/components/GrafanaComponent.js

```diff
@@ -202,7 +200,7 @@ class GrafanaComponent extends Component {
         }, self.handleError('There was an error communicating with Grafana'));
       }
 
-      handleError = (msg) => (error) => {
```

Comment:
I'm afraid it isn't how it works. I've confirmed it is a higher-order function and needs to be this way. 
See: https://github.com/layer5io/meshery/blob/c28a5355ec372568d72fca3718efd685160191f3/ui/components/GrafanaSelectionComponent.js#L170

Reply:
https://github.com/layer5io/meshery/pull/936#discussion_r418321437

Reply:
You are right, the error will show up. the problem is it will **always** show up. Since you have removed the inner function, it is executed immediately when passed into the callback instead of getting executed in the catch statement.
I'd suggest you revert this change

Reply:
Done